docs/book/src/tasks/verify-container-images.md

```diff
@@ -0,0 +1,47 @@
+# Verification of CAPI artifacts
+
```

Comment:
Probably it is better first to list the [tools](https://kubernetes.io/docs/tasks/administer-cluster/verify-signed-artifacts/#before-you-begin) needed to be able to run the command later?

Reply:
I've included `requirements` in the latest commit. Thanks :)